service/http.proxy/src/main/java/org/kaazing/gateway/service/http/proxy/HttpProxyServiceHandler.java

```diff
@@ -93,14 +112,15 @@ private boolean validateRequestPath(DefaultHttpSession acceptSession) {
     }
 
     /*
-     * Initializer for connect session. It adds the processed accept session headers
-     * on the connect session
+     * Initializer for connect session. It adds the processed accept session headers on the connect session
      */
     private static class ConnectSessionInitializer implements IoSessionInitializer<ConnectFuture> {
         private final DefaultHttpSession acceptSession;
+        private final String forwardedProperty;
 
-        ConnectSessionInitializer(DefaultHttpSession acceptSession) {
+        ConnectSessionInitializer(DefaultHttpSession acceptSession, String forwardedProperty) {
```

Comment:
@jitsni I have fetched/merged the latest changes from upstream, but this class still remains static.